tests/test_cli/test_exit_transaction_mnemonic.py

```diff
@@ -0,0 +1,99 @@
+import os
+
+from click.testing import CliRunner
+
+from staking_deposit.deposit import cli
+from staking_deposit.utils.constants import DEFAULT_EXIT_TRANSACTION_FOLDER_NAME
+
+from tests.test_cli.helpers import clean_exit_transaction_folder, read_json_file, verify_file_permission
+
+
+def test_exit_transaction_menmonic() -> None:
```

Comment:
We are only testing the happy and successful path for exit-transaction-mnemonic. It would be nice to add a few tests for the paths where it fail.